httpie/downloads.py

```diff
@@ -22,6 +22,8 @@
 
 PARTIAL_CONTENT = 206
 
+TIME_FUNC = monotonic
```

Comment:
Why the constant?

Reply:
To ease monkey patching or eventual future change. I will revert, it is not really necessary.